app/javascript/app/components/table/table-component.jsx

```diff
@@ -71,6 +79,10 @@ class SimpleTable extends PureComponent {
                     flexGrow={flexGrow}
                     cellRenderer={cell => {
```

Comment:
Looking to the future, this is the sort of thing that should be passed in by the parent. Custom render functions should be exposed by our extended components like Table so that you can pass this from the parent and the table can be reused in other projects without knowing what a TrendLine is. This can be done easily by spreading the parent props right after the customised props. No problem for now, but certainly other tables shouldn't be worrying about trendlines. What do you think?